corehq/apps/users/models.py

```diff
@@ -1247,7 +1247,8 @@ def create(cls, domain, username, password, email=None, uuid='', date='',
         else:
             couch_user.created_on = datetime.utcnow()
 
-        user_data = kwargs.get('user_data', {})
+        user_data = {'commcare_project': domain}
```

Comment:
That was from the spec. Not sure how sheel came up with it but we liked the idea of a `commcare_` namespace for all of the items that we add automatically, to limit the risk of name collisions.